change language from array to tensor

pmeier opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #61031 remove UsageError in favor of ValueError
* **#60639 change language from array to tensor**
* #60638 relax type equality constraint

Differential Revision: [D29626812](https://our.internmc.facebook.com/intern/diff/D29626812)

pmeier committed:
change language from array to tensor

[ghstack-poisoned]

pmeier committed:
Update on "change language from array to tensor"




[ghstack-poisoned]

pmeier committed:
Update on "change language from array to tensor"




[ghstack-poisoned]

pmeier committed:
Update on "change language from array to tensor"




[ghstack-poisoned]

pmeier committed:
Update on "change language from array to tensor"




[ghstack-poisoned]

mruberry reviewed (approved):
Stamped. "tensor-or-scalar-like" seems like an OK phrase to me.

mruberry commented:
@mruberry has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29626812).

pmeier committed:
Update on "change language from array to tensor"


Differential Revision: [D29626812](https://our.internmc.facebook.com/intern/diff/D29626812)

[ghstack-poisoned]

mruberry commented:
@mruberry has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D29626812).
